dojo/models.py

```diff
@@ -3848,8 +3848,8 @@ def __str__(self):
 class JIRA_Instance(models.Model):
     configuration_name = models.CharField(max_length=2000, help_text=_("Enter a name to give to this configuration"), default="")
     url = models.URLField(max_length=2000, verbose_name=_("JIRA URL"), help_text=_("For more information how to configure Jira, read the DefectDojo documentation."))
-    username = models.CharField(max_length=2000)
-    password = models.CharField(max_length=2000)
+    username = models.CharField(max_length=2000, verbose_name=_("Username/Email"), help_text=_("Username or Emailaddress, see DefectDojo documentation for more information."))
```

Comment:
```suggestion
    username = models.CharField(max_length=2000, verbose_name=_("Username/Email"), help_text=_("Username or Email Address, see DefectDojo documentation for more information."))
```

Reply:
ok. In the docs I saw Emailaddress so I assumed it must be one word in English :-)